Source/Core/InputCommon/ControllerInterface/evdev/evdev.h

```diff
@@ -15,6 +15,7 @@ namespace ciface
 namespace evdev
 {
 void Init();
+void GetDevices();
```

Comment:
The commit this change is part of should be squashed into the second one (because it's the same change logically and it would otherwise break git bisecting).